### PR TITLE
Add user project options for requester pays flag

### PR DESCRIPTION
**Pull request by shinfan**
Requester pays buckets require user project (billing) to access.

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
Add the `GcpLaunchStage.Alpha` annotation because the feature is behind a whitelist.

**Comment by shinfan** [comment]
PTAL @garrettjonesgoogle

File: google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java [hunk 1]
```diff
@@ -343,6 +344,7 @@ void populateCondition(Rule.Condition condition) {
      * Sets whether a user accessing the bucket or an object it contains should assume the transit costs
      * related to the access.
      */
+    @GcpLaunchStage.Alpha
     abstract Builder setRequesterPays(Boolean requesterPays);
```
Comment (garrettjonesgoogle) [inline google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java]: If this has already been added to a release, we can't "take back" its status as GA...

Comment (shinfan) [inline google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java]: Add the annotation as comment.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**